### PR TITLE
 [7] Develop a Contact Form Page using HTML

### PR DESCRIPTION
**Pull request by Dawodurahmonn**
## Task: `I just created a contact-form using HTML
rahh

- `Changes you made to given task`

### CODE REVIEW

File: submissions/DawoduRahmon/contact-form/contact.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+
```
Comment (emmanuelidiaghe) [inline submissions/DawoduRahmon/contact-form/contact.html:2]: Hello,

Nice work! The contact.html should be in a folder named html-forms.

**Review by emmanuelidiaghe — CHANGES_REQUESTED** [review]

File: submissions/DawoduRahmon/HTML/index.html [hunk 2]
```diff
@@ -10,8 +10,9 @@
         <h1>RILOGAD LIMITED</h1>
```
Comment (emmanuelidiaghe) [inline submissions/DawoduRahmon/HTML/index.html:10]: Please delete this index.html, I guess your product-page.html should suffice.

**Review by emmanuelidiaghe — CHANGES_REQUESTED** [review]

**Comment by Dawodurahmonn** [comment]
have done that sir, and do i need to push again?
    On Wednesday, October 7, 2020, 08:38:33 AM GMT+1, Emmanuel Idiaghe <notifications@github.com> wrote:  
 
 


@emmanuelidiaghe requested changes on this pull request.

In submissions/DawoduRahmon/HTML/index.html:
> @@ -10,8 +10,9 @@
         <h1>RILOGAD LIMITED</h1>

Please delete this index.html, I guess your product-page.html should suffice.

**Comment by emmanuelidiaghe** [comment]
Yes, you should push each time you make changes.

**Pull request closed without merging**